win/CS/HandBrakeWPF/ViewModels/SubtitlesViewModel.cs

```diff
@@ -782,6 +808,44 @@ private void AddInputSubtitles(string[] filenames)
             }
         }
 
+        private static IEnumerable<string> GetExternalSubtitlesBasedOnFileName(string directoryName, string fileName)
+        {
+            var subtitleFiles = Directory.EnumerateFiles(directoryName, $"{fileName}.*.ass").ToList();
+            subtitleFiles.AddRange(Directory.EnumerateFiles(directoryName, $"{fileName}.*.srt").ToList());

```

Comment:
There are a lot of subtitles out there that won't abide by .*. and thus not get picked up by this change.

It probably has to be ${filename}* to be most effective.